plugins/out_stackdriver/stackdriver.c

```diff
@@ -902,6 +943,11 @@ static int stackdriver_format(struct flb_config *config,
     int severity_extracted = FLB_FALSE;
     severity_t severity;
 
+    /* Parameters for insertId */
+    msgpack_object insert_id_obj;
+    insert_id_status in_status;
+    int insert_id_extracted = FLB_FALSE;
```

Comment:
declare variables at the beginning of the function

Reply:
Actually it is in the beginning.